addons/addons.json

```diff
@@ -1,4 +1,5 @@
 [
+  "play-later",
```

Comment:
Newly added addons are usually added above `"// NEW ADDONS ABOVE THIS ↑↑",`, not at the top.

Reply:
Oh, I see! I moved it to right above that comment.